### ISSUE TITLE
Video saving status

### ISSUE THREAD

**Issue by Gubaz** [saki4510t/UVCCamera#372]
Im working with usbCameraTest4 project and I need to find out when video done saving on device. is there any way to find it out ? can someone please help me with this ?

**Comment by Gubaz** [comment]
Hello , 
I mean , my goal is to show video in videoView after recording it. 
all I get is that VideoView dialog, with message: “Can't Play Video”. 
so Im thinking that , Im starting VideoView activity too early and I want to know when video done recording on sdcard to show it for user. <- maybe this is not correct way of thinking, actually I'm very confused.

**Comment by iskander-saitbatalov** [comment]
Use "usbCameraTest4" to see how you should do it correctly.

**Comment by Gubaz** [comment]
yeah I'm using "usbCameraTest4"  - I've even added open gallery functionality and video is not there 
-__-
